flink-table/flink-table-planner-blink/src/test/resources/org/apache/flink/table/planner/plan/stream/sql/TableScanTest.xml

```diff
@@ -16,36 +16,63 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 <Root>
+  <TestCase name="testDataStreamScan">
+    <Resource name="sql">
+      <![CDATA[SELECT * FROM DataStreamTable]]>
+
+    </Resource>
+    <Resource name="planBefore">
+      <![CDATA[
+LogicalProject(a=[$0], b=[$1], c=[$2])
++- LogicalTableScan(table=[[default_catalog, default_database, DataStreamTable]])
+]]>
+
+    </Resource>
+    <Resource name="planAfter">
+      <![CDATA[
+DataStreamScan(table=[[default_catalog, default_database, DataStreamTable]], fields=[a, b, c])
+]]>
+
+    </Resource>
+  </TestCase>
   <TestCase name="testTableSourceScan">
     <Resource name="sql">
       <![CDATA[SELECT * FROM MyTable]]>
+
     </Resource>
     <Resource name="planBefore">
       <![CDATA[
 LogicalProject(a=[$0], b=[$1], c=[$2])
 +- LogicalTableScan(table=[[default_catalog, default_database, MyTable, source: [TestTableSource(a, b, c)]]])
 ]]>
+
     </Resource>
     <Resource name="planAfter">
       <![CDATA[
 TableSourceScan(table=[[default_catalog, default_database, MyTable, source: [TestTableSource(a, b, c)]]], fields=[a, b, c])
 ]]>
+
```

Comment:
Remove these added blank lines.

Reply:
Plan XML files should be re-generated if it requires changes instead of updating manually.
This helps us maintain them more easily in the future. 
This blank lines is added by the XML generator,

Reply:
Fine, i'm inclined to move these changes into another single PR which only update the XML file style, we can fire the PR periodically.

After all, it is a change that are not relevant with this patch right ?

Reply:
It is related. We added a plan test in `TableScanTest`. So the corresponding XML will have one more entry. And as I said before, this file should be re-generated by the framework, not updated manually. These blank lines are generated by the framework and of course should be in a single commit.